domains.txt

```diff
@@ -617,27 +685,34 @@ sjit.co.uk
 skala.co.uk
 skillrecruit.com
 skillsearch.com
+skypath.co.uk
 smart-co.co.uk
 smartedgesolutions.co.uk
 smartsourcing.plc.uk
 snap.hr
+socode.co.uk
```

Comment:
Founded by genuine industry leaders – SoCode is a delivery and service led organisation driven by a passion for Technology and Recruitment.  

With over a decade of Technology Recruitment experience – when partnering with SoCode -  Professionalism,  Expertise and Results come as standard